public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -862,7 +883,7 @@ public ConceptAnalysis apply(ConceptAnalysis ca) {
         List<Long> countAnalysisIds = new ArrayList<>();
         countAnalysisIds.add(3300L);
         countAnalysisIds.add(3301L);
-        List<AchillesAnalysis> ehrAnalysesList = achillesAnalysisDao.findAnalysisByIds(countAnalysisIds, domainId);
+        List<AchillesAnalysis> ehrAnalysesList = achillesAnalysisDao.findAnalysisByIdsAndDomain(countAnalysisIds, domainId);
```

Comment:
remove
`List countAnalysisIds = new ArrayList<>();`
`countAnalysisIds.add(3300L); `
`countAnalysisIds.add(3301L); `
in favor of
```suggestion
        List<AchillesAnalysis> ehrAnalysesList = achillesAnalysisDao.findAnalysisByIdsAndDomain(ImmutableList.of(3300L, 3301L), domainId);
```

Reply:
Done thanks.